client.c

```diff
@@ -37,62 +37,65 @@ struct command {
+    { "quit",                   IPCQuit,                    true,  0, NULL,       "",                                         "close the window manager"},
+    { "top_gap",                IPCTopGap,                  true,  1, fn_int,     "GAP",                                      "Set the offset at the top of the monitor (usually for system bars)"},
+    { "edge_gap",               IPCEdgeGap,                 false, 4, fn_int,     "TOP BOTTOM LEFT RIGHT",                    "Set the edge gap around the monitor (must include all parameters)"},
+    { "save_monitor",           IPCSaveMonitor,             false, 2, fn_int,     "i j",                                      "Associate the ith monitor to the jth workspace"},
+    { "smart_place",            IPCSmartPlace,              true,  1, fn_bool,    "true/false",                               "Determine whether or not newly placed windows should be placed in the largest available space."},
+    { "draw_text",              IPCDrawText,                true,  1, fn_bool,    "true/false",                               "Determine whether or not text should be draw in title bars"},
+    { "edge_lock",              IPCEdgeLock,                true,  1, fn_bool,    "true/false",                               ""},
+    { "set_font",               IPCSetFont,                 false, 1, fn_font,    "FONT_NAME",                                "Set the name of the font to use (e.g. set_font 'xft:Rubik:size=10:style=Bold')"},
+    { "json_status",            IPCJSONStatus,              true,  1, fn_bool,    "true/false",                               "Determine whether or not BERRY_WINDOW_STATUS returns JSON formatted text."},
+    { "manage",                 IPCManage,                  true,  1, fn_str,     "Dialog|Toolbar|Menu|Splash|Utility",       "Set berry to manage clients of the above type. Clients which are managed will be given decorations and are movable by the window manager. This is not retroactive for current clients. Only Toolbars and Splahes are not handled by default."},
+    { "unmanage",               IPCUnmanage,                true,  1, fn_str,     "Dialog|Toolbar|Menu|Splash|Utility",       "Set berry to not manage clients of the above type."},
+    { "decorate_new",           IPCDecorate,                true,  1, fn_bool,    "true/false",                               "Determine whether or not new windows are decorated by default"},
+    { "name_desktop",           IPCNameDesktop,             false, 2, fn_int_str, "DESKTOP NAME",                             "Name the ith desktop name. Used with _NET_DESKTOP_NAMES."},
+    { "move_mask",              IPCMoveMask,                true,  1, fn_mask,    "shift|lock|ctrl|mod1|mod2|mod3|mod4|mod5", ""},
+    { "resize_mask",            IPCResizeMask,              true,  1, fn_mask,    "shift|lock|ctrl|mod1|mod2|mod3|mod4|mod5", ""},
```

Comment:
Cool :+1: I was looking for something like this in #175 

But can you update the manual page too? https://github.com/JLErvin/berry/blob/master/berryc.1#L167